src/components/Page/PageWorkArea.jsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import { spacing05, spacing07 } from '@carbon/layout';
 
 const StyledDiv = styled.div`
   width: 100%;
-  padding: 2rem 1rem;
+  padding: ${spacing05} ${spacing07};
```

Comment:
should these be flipped?
```suggestion
  padding: ${spacing07} ${spacing05};
```

Reply:
Yeah these should be flipped @soniadeveloper